README.md

```diff
@@ -70,6 +70,15 @@ GaEvents.Event.adapter = function() {
 }
 ```
 
+### Optional consent settings
+
+Events are flushed immediatly by default. If you need to wait for user consent
+you can set `GaEvents.Event.require_user_consent = true`.
+
+With `require_user_consent` enabled all events are buffered until
+`GaEvents.Event.user_consent_given = true` is set. Events are flushed als soon
+als `GaEvents.Event.flush()` is called.
```

Comment:
TODO: Hier + Zeile drüber: "as", nicht "als"

Reply:
fixed in b6c25432815a0d7ae211f7bec95bf6f1f20de882